doc/develop/fixing_data.rst

```diff
@@ -1,32 +1,39 @@
 .. _fixing_data:
 
 ***********
-Dataset fix
+Fixing data
 ***********
 
-Some (model) datasets contain (known) errors that would normally prevent them
-from being processed correctly by the ESMValCore. The errors can be in
+The baseline case for ESMValCore input data is CMOR fully compliant
+data that is read using
+`Iris load_raw function <https://scitools-iris.readthedocs.io/en/latest/generated/api/iris.html#iris.load_raw>`_.
```

Comment:
Thanks to intersphinx, we can write this much more compact as
```suggestion
data that is read using Iris' :func:`iris:iris.load_raw`.
```
For more information see [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) and [roles](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects).